util.go

```diff
@@ -154,11 +155,17 @@ func trimeSchemaRefLinkPrefix(ref string) string {
 	return strings.TrimPrefix(ref, "#/components/schemas/")
 }
 
-func genSchemeaObjectID(pkgName, typeName string) string {
-	typeNameParts := strings.Split(typeName, ".")
-	pkgName = replaceBackslash(pkgName)
+func genSchemeaObjectID(pkgName, typeName string, p *parser) string {
```

Comment:
Nit. Any chance we can fix this typo while we're at it? I'm assuming it's supposed to be `genSchemaObjectId`